pkg/deploy/generator/resources_rp.go

```diff
@@ -829,7 +831,9 @@ fi
 SECRET_NAME="rp-\${COMPONENT}"
 NEW_CERT_FILE="\$TEMP_DIR/\$COMPONENT.pem"
 for attempt in {1..5}; do
-  az keyvault secret download --file \$NEW_CERT_FILE --id "https://$KEYVAULTPREFIX-svc.vault.azure.net/secrets/\$SECRET_NAME" && break
+  echo "Keyvault Prefix: $KEYVAULTPREFIX"
+  echo "Secret Name: $SECRET_NAME"
```

Comment:
Is this necessary to echo?

Reply:
It can be removed, was put in to help debug a deployment.

Reply:
Please remove echoing the secret name and keyvault prefix: @coreyperkins